src/twisted/trial/_dist/test/test_worker.py

```diff
@@ -272,9 +274,10 @@ def stopTest(self, test):
         return d.addCallback(self.assertIdentical, result)
 
 
-class FakeAMProtocol(AMP):
+class SpyDataLocalWorkerAMP(LocalWorkerAMP):
```

Comment:
I went with using `LocalWorkerAMP` as much as possible to test the real deal and reduce code duplication in tests.